test/watch.js

```diff
@@ -66,14 +83,11 @@ describe('gulp', function() {
 
       createTempFile(tempFile);
 
-      var watcher = gulp.watch(relFile, {debounceDelay: 5, cwd: cwd})
-        .on('change', function(evt) {
-          should.exist(evt);
-          should.exist(evt.path);
-          should.exist(evt.type);
-          evt.type.should.equal('changed');
-          evt.path.should.equal(path.resolve(tempFile));
-          watcher.end();
+      var watcher = gulp.watch(relFile, {cwd: cwd})
+        .on('change', function(filepath) {
+          should.exist(filepath);
+          path.resolve(cwd, filepath).should.equal(path.resolve(tempFile));
```

Comment:
This one was particular tricky, as chokidar spits out relative file paths instead of absolute ones.